cmd/podman/containers/run.go

```diff
@@ -101,7 +103,19 @@ func run(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-
+	if rootless.IsRootless() {
+		if uid, err := strconv.ParseInt(cliVals.User, 10, 64); err == nil {
```

Comment:
There is a good chance that the user entered a username, not a string - this would be a lot more useful if it ran as part of libpod Init() after the container is mounted and know what the UID is.

Reply:
I think it might be better to do a double check.  Since the second check has to happen after the image has been pulled.  If the user gives us a UID, then we can check right away.  If they give us one with a username, we can report back that it will not work inside of the user namespace.